tests/data/eq-mwss.json

```diff
@@ -1,112 +1,112 @@
-   "tx_id":"40e659ec-013f-4993-9a31-ec1e0ad37888",
-   "data":{  
-      "133w":"Weekly",
-      "134f":"Fortnightly",
-      "130":"Calendar monthly",
-      "131":"Four weekly",
-      "132":"Five weekly",
-      "40":"5",
-      "50":"445566",
-      "60":"112233",
-      "70":"334455",
-      "80":"556677",
-      "90w":"Yes",
-      "91w":"redundancies",
-      "92w1":"more temporary staff",
-      "92w2":"fewer temporary staff",
-      "94w1":"more overtime",
-      "94w2":"less overtime",
-      "95w":"pay increase",
-      "96w":"industrial action",
-      "97w":"other",
-      "100":"50",
-      "110":"01/02/2016",
-      "120":"25",
-      "300w":"Pipe mania",
-      "40f":"10",
-      "50f":"123123",
-      "60f":"456456",
-      "80f":"101010",
-      "70f":"789789",
-      "90f":"Yes",
-      "91f":"redundancies",
-      "92f1":"more temporary staff",
-      "92f2":"fewer temporary staff",
-      "94f1":"more overtime",
-      "94f2":"less overtime",
-      "95f":"pay increase",
-      "96f":"industrial action",
-      "97f":"other",
-      "110f":"02/03/2017",
-      "100f":"60",
-      "120f":"30",
-      "300f":"Gas leak",
-      "140m":"20",
-      "151":"321321",
-      "181":"999999",
-      "171":"121212",
-      "190m":"Yes",
-      "191m":"redundancies",
-      "192m1":"more temporary staff",
-      "192m2":"fewer temporary staff",
-      "194m1":"more overtime",
-      "194m2":"less overtime",
-      "195m":"pay increase",
-      "196m":"industrial action",
-      "197m":"other",
-      "210":"03/04/2018",
-      "200":"70",
-      "220":"40",
-      "300m":"copper pipe",
-      "140w4":"30",
-      "152":"98765",
-      "172":"443322",
-      "182":"767676",
-      "190w4":"Yes",
-      "191w4":"redundancies",
-      "192w41":"more temporary staff",
-      "192w42":"fewer temporary staff",
-      "194w41":"more overtime",
-      "194w42":"less overtime",
-      "195w4":"pay increase",
-      "196w4":"industrial action",
-      "197w4":"other",
-      "200w4":"80",
-      "210w4":"04/05/2019",
-      "220w4":"50",
-      "300w4":"solder joint",
-      "140w5":"40",
-      "153":"13134",
-      "173":"989",
-      "183":"9112",
-      "190w5":"Yes",
-      "191w5":"redundancies",
-      "192w51":"more temporary staff",
-      "192w52":"fewer temporary staff",
-      "194w51":"more overtime",
-      "194w52":"less overtime",
-      "195w5":"pay increase",
-      "196w5":"industrial action",
-      "197w5":"other",
-      "200w5":"90",
-      "220w5":"60",
-      "210w5":"05/06/2020",
-      "300w5":"drill hole",
-      "300":"flux clean"
-   },
-   "type":"uk.gov.ons.edc.eq:surveyresponse",
-   "version":"0.0.1",
-   "metadata":{  
-      "user_id":"K5O86M2NU1",
-      "ru_ref":"12346789012A"
-   },
-   "submitted_at":"2017-03-01T14:25:46.101447+00:00",
-   "collection":{  
-      "period":"201605",
-      "exercise_sid":"82R1VDWN74",
-      "instrument_id":"0005"
-   }
-}
+{
+    "origin": "uk.gov.ons.edc.eq",
```

Comment:
Do we need to change this file?

Reply:
Yes - my formatter is the one true formatter.